src/test/obj_ulog_size/obj_ulog_size.c

```diff
@@ -72,6 +73,12 @@
 #define REDO_OVERFLOW ((size_t)((LANE_REDO_EXTERNAL_SIZE\
 		/ TX_INTENT_LOG_ENTRY_OVERHEAD) + 1))
 
+#if CACHELINE_SIZE > 64
```

Comment:
This test includes internals, so you could do:
`#define APPEND_SIZE SIZEOF_ALIGNED_ULOG(64)`.
Should do roughly the same thing.

Reply:
Thanks, I didn't notice this.